src/lib/prompts.js

```diff
@@ -0,0 +1,244 @@
+        if (isNaN(val)) return red('Fee must be a number.');
+        if (val < 0) return red("Fee can't be negative.");
+        return true;
+      },
+      result: (val) => val.trim().replace(/ /, ''),
+    },
+  ],
+
+  initialFeepayerPrompts: (
+    defaultFeepayerAlias,
+    defaultFeepayerAddress,
+    isFeepayerCached
+  ) => [
+    {
+      type: 'select',
+      name: 'feepayer',
+      choices: [
+        {
+          name: `Recover fee payer account from an existing base58 private key`,
+          value: 'recover',
+        },
+        {
+          name: `Create a new fee payer key pair
+  NOTE: the private key will be stored in plain text on this computer.`,
+          value: 'create',
+        },
+      ],
+      message: (state) => {
+        const style = state.submitted && !state.cancelled ? green : reset;
+        return style('Choose an account to pay transaction fees:');
+      },
+
+      skip() {
+        return isFeepayerCached; // The prompt is only displayed if a feepayor has not been previously cached
+      },
+      result() {
+        // Workaround for a bug in enquirer that returns the first value of choices when the
+        // question is skipped https://github.com/enquirer/enquirer/issues/340 .
+        // This returns the previous prompt value if prompt is skipped.
+        if (isFeepayerCached) {
+          return this.state.answers.feepayer;
+        }
+        return this.focused.value;
+      },
+    },
+    {
+      type: 'select',
+      name: 'feepayer',
+      choices: [
+        {
+          name: chalk`Use stored account {bold ${defaultFeepayerAlias}} (public key: {bold ${defaultFeepayerAddress}}) `,
+          value: 'defaultCache',
+        },
+        {
+          name: 'Use a different account (select to see options)',
+          value: 'other',
+        },
+      ],
+      message: (state) => {
+        const style = state.submitted && !state.cancelled ? green : reset;
+        return style('Choose an account to pay transaction fees:');
+      },
+      skip() {
+        return !isFeepayerCached; // Only display this prompt question if feeypayer is cached
+      },
+      result() {
+        // Workaround for a bug in enquirer that returns the first value of choices when the
+        // question is skipped https://github.com/enquirer/enquirer/issues/340 .
+        // This returns the previous prompt value if prompt is skipped.
+
+        if (!isFeepayerCached) {
+          return this.state.answers.feepayer;
+        }
+        return this.focused.value;
+      },
+    },
+  ],
+  recoverFeepayerPrompts: (cachedFeepayerAliases) => [
+    {
+      type: 'input',
+      name: 'feepayerAliasName',
+      message: (state) => {
+        const style = state.submitted && !state.cancelled ? green : reset;
+        return style('Create an alias for this account');
+      },
+      validate: async (val) => {
+        val = val.toLowerCase().trim().split(' ').join('-');
```

Comment:
It will be good to replace the groups of whitespaces with single dash. We can use RegExp like `/\s{1,}/g`.